content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/fedex.rst

```diff
@@ -0,0 +1,173 @@
+account holder's name in the top-right corner of the screen, and select :menuselection:`My Profile`
+from the drop-down menu.
+
+On the profile page, click :guilabel:`Account Management` on the left side of the screen. The
+account number is displayed on this screen.
+
+Once the password and account number are determined, enter them in the :guilabel:`Password` and
+:guilabel:`Account Number` fields on the :guilabel:`Fedex Configuration` tab of the shipping method.
+
+Developer Key and Meter Number
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+A *developer key* is used to integrate a FedEx account with an external service, like the Odoo
+Inventory app. A *meter number* is a unique ID number used by FedEx to identify negotiated shipping
+rates for each account.
+
+To get a developer key and meter number, begin by navigating to FedEx's `Developer Resource Center
+<https://www.fedex.com/en-us/developer/web-services.html>`_. Then, click on the :guilabel:`FedEx Web
+Services` drop-down menu.
+
+Click on :guilabel:`Get Test Key` to start the process of getting a developer key and meter number
+which can be used to configure a shipping method for testing purposes.
+
+Click on :guilabel:`Get Production Key` to start the process of getting a developer key and meter
+number which can be used to configure a shipping method that generates real shipments with FedEx.
+
+After clicking either option, follow the instructions until the :guilabel:`Confirmation` screen is
+reached. This screen displays the developer key and meter number.
+
+Once the developer key and meter number are determined, enter them in the :guilabel:`Developer Key`
+and :guilabel:`Meter Number`fields on the :guilabel:`Fedex Configuration` tab of the shipping
+method.
+
+Shipping details
+~~~~~~~~~~~~~~~~
+
+The main section of the :guilabel:`Fedex Configuration` tab includes a number of additional fields
+used provide information about the shipping method:
+
+- :guilabel:`Fedex Service Type`: the FedEx service used to ship a package.
+- :guilabel:`Fedex Drop-Off Type`: the method for getting a package into FedEx's possession.
+- :guilabel:`Fedex Package Type`: the type of package used for the shipping method.
+- :guilabel:`Package Weight Unit`: the unit of measure used to weigh packages.
+- :guilabel:`Package Length Unit`: the unit of measure used to determine the dimensions of packages.
+- :guilabel:`Label Type`: the type of shipping label used for packages.
+- :guilabel:`Label Format`: the file format used by Odoo to generate shipping labels.
+- :guilabel:`Commercial Invoice Type`: the dimensions and type of the paper used to print invoices.
+
+.. important::
+   The options that should be selected on the :guilabel:`Configuration` tab of a shipping method
+   depend on the negotiated shipping services of the associated FedEx account. To confirm the
+   available services for a FedEx account, visit the :guilabel:`Account Management` page after
```

Comment:
```suggestion
   available services for a FedEx account, visit the *Account Management* page after
```
opting for italics here because this page is not currently being shown on the screen/UI